pennylane_ionq/ops.py

```diff
@@ -61,28 +61,34 @@ class GPI2(Operation):  # pylint: disable=too-few-public-methods
 
 
 class MS(Operation):  # pylint: disable=too-few-public-methods
-    r"""MS(phi0, phi1, wires)
-    2-qubit entanlging MS gate.
+    r"""MS(phi0, phi1, theta=0.25, wires)
+    2-qubit entangling MS gate.
 
     .. math::
 
-       MS(\phi_{0}, \phi_{1}) =
+       MS(\phi_{0}, \phi_{1}, \theta) =
             \frac{1}{\sqrt{2}}\begin{pmatrix}
-                1 & 0 & 0 & -i e^{-2 \pi i(\phi_{0}+\phi_{1})} \\
-                0 & 1 & -i e^{-2 \pi i (\phi_{0}-\phi_{1})} & 0 \\
-                0 & -i e^{2 \pi i(\phi_{0}-\phi_{1})} & 1 & 0 \\
-                -i e^{2 \pi i(\phi_{0}+\phi_{1})} & 0 & 0 & 1
+                \cos(\theta / 2) & 0 & 0 & -i e^{-2 \pi i(\phi_{0}+\phi_{1})} \\
+                0 & \cos(\theta / 2) & -i e^{-2 \pi i (\phi_{0}-\phi_{1})} & 0 \\
+                0 & -i e^{2 \pi i(\phi_{0}-\phi_{1})} & \cos(\theta / 2) & 0 \\
+                -i e^{2 \pi i(\phi_{0}+\phi_{1})} & 0 & 0 & \cos(\theta / 2)
             \end{pmatrix}
+
     Args:
-        phi0 (float): phase of the first qubit :math:`\phi`
-        phi1 (float): phase of the second qubit :math:`\phi`
+        phi0 (float): phase of the first qubit :math:`\phi_0`
+        phi1 (float): phase of the second qubit :math:`\phi_1`
+        theta (float): entanglement ratio of the qubits :math:`\theta \in [0, 0.25]`, defaults to 0.25
         wires (Sequence[int]): the subsystems the operation acts on
     """
 
-    num_params = 2
+    num_params = 3
     num_wires = 2
+    par_domain = "R"
```

Comment:
I think this was removed quite some time ago.   Does anything in the ionq plugin need it?

```suggestion
```

Reply:
no nothing needs this - i saw in it in some docs but will remove :)